workers/tests/integrationtests/test_oqt.py

```diff
@@ -14,157 +20,114 @@
 class TestOqt(unittest.TestCase):
     def setUp(self):
         # Heidelberg
+        self.name = "GhsPopComparisonBuildings"
+        self.report_name = "SimpleReport"
+        self.layer_name = "building_count"
         self.dataset = "regions"
         self.feature_id = "3"
         self.fid_field = "ogc_fid"
         self.feature = asyncio.run(
             db_client.get_feature_from_db(self.dataset, feature_id=self.feature_id)
         )
 
-    @oqt_vcr.use_cassette()
-    def test_create_indicator_feature(self):
-        """Test creating indicator from scratch."""
-        # From scratch
-        indicator = asyncio.run(
-            oqt.create_indicator(
-                "GhsPopComparisonBuildings", "building_count", feature=self.feature
-            )
-        )
+    def run_tests(self, indicator):
         self.assertIsNotNone(indicator.result.label)
         self.assertIsNotNone(indicator.result.value)
         self.assertIsNotNone(indicator.result.description)
         self.assertIsNotNone(indicator.result.svg)
 
     @oqt_vcr.use_cassette()
-    def test_create_indicator_invalid_dataset(self):
-        with self.assertRaises(ValueError):
-            asyncio.run(
-                oqt.create_indicator(
-                    "GhsPopComparisonBuildings",
-                    "building_count",
-                    dataset="foo",
-                    feature_id=self.feature_id,
-                )
-            )
-
-    @oqt_vcr.use_cassette()
-    def test_create_indicator_dataset_invalid_fid_field(self):
-        with self.assertRaises(ValueError):
-            asyncio.run(
-                oqt.create_indicator(
-                    "GhsPopComparisonBuildings",
-                    "building_count",
-                    dataset=self.dataset,
-                    feature_id=self.feature_id,
-                    fid_field="foo",
-                )
-            )
+    def test_create_indicator_bpolys(self):
+        """Test creating indicator from scratch."""
+        param = IndicatorBpolys(
+            name=self.name,
```

Comment:
`param` sounds to me like only one parameter is included, I would prefer `parameters` (or if you want a shorter version maybe `params`). The same applies to the other tests in this file.

Reply:
I changed all occurrences of param to parameters.

Reply:
Oh, I missed that. I only changed it in the ohsome_quality_analyst directory not in the test directory. But now its also changed for the tests.